scm-webapp/src/test/java/sonia/scm/security/BearerRealmTest.java

```diff
@@ -62,7 +65,6 @@
   @Mock
   private AccessTokenResolver accessTokenResolver;
 
-  @InjectMocks
```

Comment:
Do we need no longer mock injection?

Reply:
No, since the realm is initialized in the beforeEach anyway.